[REG 1.20 -> 1.21] Cannot use on node/ts without esModulesInterop

Hi,

Starting from version 1.21.0, we cannot use this package with typescript because it requires `esModulesInterop`, which we don't use (and it breaks our build).

```
node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/common.browser/RestMessageAdapter.d.ts:2:8 - error TS1259: Module '"node_modules/@types/bent/index"' can only be default-imported using the 'esModuleInterop' flag

2 import bent from "bent";
         ~~~~

  node_modules/@types/bent/index.d.ts:77:1
    77 export = bent;
       ~~~~~~~~~~~~~~
    This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.
```

--- a/src/common.browser/RestMessageAdapter.ts
+++ b/src/common.browser/RestMessageAdapter.ts
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-import bent, { BentResponse } from "bent";
+import bent, { BentResponse, RequestBody } from "bent";
 import {
     ArgumentNullError,
     Deferred
@@ -82,7 +82,7 @@ export class RestMessageAdapter {
         method: RestRequestType,
         uri: string,
         queryParams: { [key: string]: any } = {},
-        body: bent.RequestBody = null,
+        body: any = null,
         binaryBody: Blob | Buffer = null,
         ): Promise<IRestResponse> {
 
@@ -111,7 +111,7 @@ export class RestMessageAdapter {
             });
         };
 
-        const send = (postData: bent.RequestBody): void => {
+        const send = (postData: RequestBody): void => {
             const sendRequest = bent(uri, requestCommand, this.privHeaders, 200, 201, 202, 204, 400, 401, 402, 403, 404);
             const params = this.queryParams(queryParams) === "" ? "" : `?${this.queryParams(queryParams)}`;
             sendRequest(params, postData).then( async (data: BentResponse): Promise<void> => {
@@ -140,7 +140,7 @@ export class RestMessageAdapter {
             this.privHeaders["content-type"] = contentType;
             this.privHeaders["Content-Type"] = contentType;
             if (typeof (Blob) !== "undefined" && binaryBody instanceof Blob) {
-                blobToArrayBuffer(binaryBody).then( (res: bent.RequestBody): void => {
+                blobToArrayBuffer(binaryBody).then( (res: RequestBody): void => {
                     send(res);
                 }).catch((error: any): void => {
                     responseReceivedDeferral.reject(error as string);
@@ -153,7 +153,7 @@ export class RestMessageAdapter {
                 this.privHeaders["content-type"] = "application/json";
                 this.privHeaders["Content-Type"] = "application/json";
             }
-            send(body);
+            send(body as RequestBody);
         }
         return responseReceivedDeferral.promise;
     }

--- a/tests/OCSPCacheTests.ts
+++ b/tests/OCSPCacheTests.ts
@@ -142,7 +142,7 @@ test("Test OCSP Staple", async (done: jest.DoneCallback) => {
     done();
 });
 
-test("Test OCSP Basic", async (done: jest.DoneCallback) => {
+test.skip("Test OCSP Basic", async (done: jest.DoneCallback) => {
     // tslint:disable-next-line:no-console
     console.info("Name: Test OCSP Basic");
 
@@ -153,7 +153,7 @@ test("Test OCSP Basic", async (done: jest.DoneCallback) => {
     done();
 });
 
-test("Test OCSP 2nd request mem cache hit.", async (done: jest.DoneCallback) => {
+test.skip("Test OCSP 2nd request mem cache hit.", async (done: jest.DoneCallback) => {
     // tslint:disable-next-line:no-console
     console.info("Name: Test OCSP 2nd request mem cache hit.");
 
@@ -174,7 +174,7 @@ test("Test OCSP 2nd request mem cache hit.", async (done: jest.DoneCallback) => 
     done();
 });
 
-test("Test OCSP expirey refreshes.", async (done: jest.DoneCallback) => {
+test.skip("Test OCSP expirey refreshes.", async (done: jest.DoneCallback) => {
     // tslint:disable-next-line:no-console
     console.info("Name: Test OCSP expirey refreshes.");
 
@@ -216,7 +216,7 @@ test.skip("Test OCSP expirey approaching refreshes.", async (done: jest.DoneCall
     done();
 });
 
-test("Test OCSP invalid cert refreshes.", async (done: jest.DoneCallback) => {
+test.skip("Test OCSP invalid cert refreshes.", async (done: jest.DoneCallback) => {
     // tslint:disable-next-line:no-console
     console.info("Name:  invalid cert refreshes.");
 

